lib/utils.js

```diff
@@ -0,0 +1,6 @@
+export const slugify = (str) => {
+  return str
+    .toLowerCase()
+    .replace(/[^\w ]+/g, "")
+    .replace(/ +/g, "-");
```

Comment:
we probably also want to filter out all special and invalid characters (things like `=!@#$%^&/[]'":;,...`), and there's probably a library we can use for that. it would be nice to also add it at the client side to they can't even input invalid things. 

once some of this is finalized i can work on that and maybe checking for things like name/slug are unique/valid + giving feedback in the form.